RELEASE_NOTES.txt

```diff
@@ -1,6 +1,88 @@
 Apache Ignite Release Notes
 ===========================
 
+
+Apache Ignite In-Memory Database and Caching Platform 2.8.1
```

Comment:
Let's rename to "Apache Ignite In-Memory Computing Platform" as per the website. The rest looks good, thanks!

Reply:
fixed.